package.json

```diff
@@ -120,7 +120,7 @@
     "@gravity-ui/prettier-config": "^1.0.1",
     "@gravity-ui/stylelint-config": "^1.0.0",
     "@gravity-ui/tsconfig": "^1.0.0",
-    "@gravity-ui/uikit": "^5.4.1",
+    "@gravity-ui/uikit": "^5.9.1",
```

Comment:
why?

Reply:
It's remove `bem-cn-lite` from dependencies. Old version worked with `bem-cn-lite`. I changed it. New version work with `@bem-react/classname`.